APPROPS.md

```diff
@@ -0,0 +1,63 @@
+# Approps Overview
```

Comment:
```suggestion
# AppProps Overview
```
Also, the file should be called `APPPROPS.md` or maybe `APP_PROPS.md`.

Reply:
I renamed the file to` APP_PROPS.md`